ui/src/submissions/conferences/containers/__tests__/ExistingConferencesAlertContainer.test.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+
+import ExistingConferencesAlertContainer from '../ExistingConferencesAlertContainer';
+import { EXISTING_CONFERENCES_NS } from '../../../../reducers/search';
+import { SEARCH_QUERY_UPDATE } from '../../../../actions/actionTypes';
+import { getStore } from '../../../../fixtures/store';
+import { RANGE_AGGREGATION_SELECTION_SEPARATOR } from '../../../../common/constants';
+
+describe('ExistingConferencesAlertContainer', () => {
+  it('dispatches SEARCH_QUERY_UPDATE', () => {
```

Comment:
`dispatches SEARCH_QUERY_UPDATE initially`

we have to `when dates is changed` but that's also tricky because of a reason I mentioned above. Leave a similar `TODO` for now :/, Perhaps our approach is odd or possibly wrong, that's why it becomes harder to test. I'll check tomorrow, how we can do this.